webchat/auth/authService.js

```diff
@@ -175,6 +182,10 @@
             return false;
         };
 
+        service.getChatClient = () => {
```

Comment:
why a get function if you have already defined set property that returns the same object?